exercism/wordy/wordy.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  WordProblem: class WordProblem {
+    constructor(input){
+      this.input = input,
+      this.operands = {
```

Comment:
These are called operators - operands are the "arguments" provided for operators.  (In the expression 3 + 5, 3 and 5 are operands, and + is an operator.)